docs/application/dotnet/guides/nui/imageview.md

```diff
@@ -1,39 +1,101 @@
 
-**Figure: Image view example**
+## Create an ImageView
 
-![Image view example](media/ImageView.png)
+To create an image view:
 
-<a name="usage"></a>
-## Creating an Image View
+**Figure: ImageView**
 
-To create an image view:
+ ![Image View](media/ImageView.png)
 
-1.  Create an instance of the [Tizen.NUI.BaseComponents.ImageView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.ImageView.html) class and define the image file as a parameter:
+1.  To use the `ImageView` class, add the following namespaces:
 
+    ```csharp
+    using Tizen.NUI;
+    using Tizen.NUI.BaseComponents;
     ```
-    imageView = new ImageView(DirectoryInfo.Resource + "images/gallery-3.jpg");
+
+2.  It is assumed that image is in the resources directory. Create an instance of the `ImageView` class and pass the path to image file as an constructor argument:
+
+    ```csharp
+    ImageView imageView = new ImageView(DirectoryInfo.Resource + "your-image.jpg");
+    Window.Instance.Add(imageView);
     ```
 
-    You can also create the `Tizen.NUI.BaseComponents.ImageView` class instance separately and define the file path by setting its `ResourceUrl` property:
+    You can also create the `ImageView` class instance separately, and define the file path by setting its `ResourceUrl` property:
 
+    ```csharp
+    imageView = new ImageView();
+    imageView.ResourceUrl = DirectoryInfo.Resource + "your-image.jpg";
+    Window.Instance.Add(imageView);
     ```
+
+2.  To change the image path after the instance has been created, use the `SetImage()` method of the `ImageView` class:
+
+    ```csharp
     imageView = new ImageView();
-    imageView.ResourceUrl = DirectoryInfo.Resource + "images/gallery-3.jpg";
+    imageView.SetImage(DirectoryInfo.Resource + "your-image.jpg");
+    Window.Instance.Add(imageView);
     ```
+## Manipulating an Image View
+
+
+**Figure: Manipulate ImageView**
 
-2.  To change the image path after the instance has been created, use the `SetImage()` method of the `Tizen.NUI.BaseComponents.ImageView` class:
+ ![Manipulated Image View](media/ImageViewManipulated.png)
 
+1.  Create object, set `Size2D` property, and display it at the center of the window by setting `ParentOrigin` and `Position2D` properties:
+
+    ```csharp
+    ImageView imageView = new ImageView(DirectoryInfo.Resource + "your-image.jpg");
+    imageView.Size2D = new Size2D(400, 400);
+    imageView.ParentOrigin = ParentOrigin.Center;
+    imageView.Position2D = new Position2D(-200, -200);
+    Window.Instance.Add(imageView);
     ```
-    imageView.SetImage(DirectoryInfo.Resource + "images/gallery-4.jpg");
+
+2.  Use `PixelArea` property to apply the zoom effect:
+
+    ```csharp
+    imageView.PixelArea = new RelativeVector4(0.25f, 0.25f, 0.5f, 0.5f);
     ```
 
-<a name="properties"></a>
-## Image View Properties
+3.  To rotate image by 45 degrees, set the `Orientation` property as follow:
```

Comment:
3.  To rotate the image by 45 degrees, set the `Orientation` property as follows: